mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -272,6 +272,42 @@ def test_ascii(tmpdir):
     assert_allclose(times_new, times)
 
 
+def test_ch_names_comma(tmpdir):
+    """Test that channel names containing commas are properly read."""
+    # commas in BV are encoded as \1
+    replace_dict = {
+        "Ch4=F4,,0.5,µV": r"Ch4=F4\1foo,,0.5,µV",
+        "4     F4": r"4     F4,foo",
+    }
+
+    # Copy existing vhdr file to tmpdir and manipulate to contain
+    # a channel with comma
+    for src, dest in zip((vhdr_path, vmrk_path, eeg_path),
+                         ('test.vhdr', 'test.vmrk', 'test.eeg')):
+        shutil.copyfile(src, tmpdir / dest)
+
+    comma_vhdr = tmpdir / 'test.vhdr'
+    with open(comma_vhdr, 'r') as fin:
+        lines = fin.readlines()
+
+    new_lines = []
+    for line in lines:
+        for to_replace, replacement in replace_dict.items():
+            if to_replace in line:
+                new = line.replace(to_replace, replacement)
+                new_lines.append(new)
+                break
+        else:
+            new_lines.append(line)
```

Comment:
The tests seem to be failing on Windows. Perhaps the manipulation of the test file prior to the test run is not working correctly on Windows?

Maybe this suggestion can help us to be sure:

```suggestion
    new_lines = []
    nperformed_replacements = 0
    for line in lines:
        for to_replace, replacement in replace_dict.items():
            if to_replace in line:
                new = line.replace(to_replace, replacement)
                new_lines.append(new)
                nperformed_replacements += 1
                break
        else:
            new_lines.append(line)
    assert nperformed_replacements == len(replace_dict)
```